### PR TITLE
add cluster-reg-namespace and hostNetwork args

### PR DESCRIPTION
**Pull request by DanielXLee**
Signed-off-by: DanielXLee <lixin8611@gamil.com>

### CODE REVIEW

File: charts/clusternet-agent/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ helm install clusternet-agent -n clusternet-system --create-namespace \
   --set parentURL=PLEASE-CHANGE-ME \
   --set registrationToken=PLEASE-CHANGE-ME \
   --set extraArgs.cluster-reg-name=PLEASE-CHANGE-ME \
+  --set extraArgs.cluster-reg-namespace=PLEASE-CHANGE-ME \
```
Comment (dixudx) [inline charts/clusternet-agent/README.md:24]: This is optional. We should make it like a required parameter.

Comment (DanielXLee) [reply charts/clusternet-agent/README.md:24]: should not?

Comment (dixudx) [reply charts/clusternet-agent/README.md:24]: Yeah, better not make it mandatory. This would mislead users.

Comment (DanielXLee) [reply charts/clusternet-agent/README.md:24]: ok, I'll remove it

**Review by dixudx — APPROVED** [review]

**Pull request merged**